op-node/rollup/derive/l1_block_info.go

```diff
@@ -54,18 +54,17 @@ func (info *L1BlockInfo) MarshalBinary() ([]byte, error) {
 	copy(data[offset:offset+32], info.BlockHash.Bytes())
 	offset += 32
 	binary.BigEndian.PutUint64(data[offset+24:offset+32], info.SequenceNumber)
-	//offset += 32
-	//copy(data[offset+12:offset+32], info.BatcherAddr[:])
-	//offset += 32
-	//copy(data[offset:offset+32], info.L1FeeOverhead[:])
-	//offset += 32
-	//copy(data[offset:offset+32], info.L1FeeScalar[:])
+	offset += 32
+	copy(data[offset+12:offset+32], info.BatcherAddr[:])
+	offset += 32
+	copy(data[offset:offset+32], info.L1FeeOverhead[:])
+	offset += 32
+	copy(data[offset:offset+32], info.L1FeeScalar[:])
 	return data, nil
 }
 
 func (info *L1BlockInfo) UnmarshalBinary(data []byte) error {
-	//if len(data) != 4+32*8 {  // TODO updated block info contract
-	if len(data) != 4+32+32+32+32+32 {
+	if len(data) != 4+32*8 {
```

Comment:
I think it makes sense to define `4+32*8` as a constant and reuse it in this file